l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -109,3 +113,27 @@ def open(self, mode: str = "r", cached: bool = True, cache_size_bytes: int = int
         self.frames = self.root[FRAME_ARRAY_KEY]
         self.agents = self.root[AGENT_ARRAY_KEY]
         self.scenes = self.root[SCENE_ARRAY_KEY]
+
+    def __repr__(self) -> str:
+        fields = [
+            "Num Scenes",
+            "Num Frames",
+            "Num Agents",
+            "Total Time (hr)",
+            "Avg Frames per Scene",
+            "Avg Agents per Frame",
+            "Avg Scene Time (sec)",
+        ]
+        values = [
+            len(self.scenes),
+            len(self.frames),
+            len(self.agents),
+            len(self.frames) / (10 * 3600),
```

Comment:
How about a param for the sampling frequency?

Reply:
that information is not in the `.zarr` actually :(  It would be possible to compute it by reading the frames `timestamp`, but getting an average may slow the print function down a lot, because we need to read all the frames. We can also read the first N frames and compute their average, but I fear it won't be very indicative

Reply:
@lucabergamini Then a hard coded constant param for it, Like `FRAME_RATE=10  # Hz`

Reply:
Ideally it would be great if we could read that out, but yes @yye-lyft , I meant that: In case we later change to a sampling frequency, we only have to modify it in one position.

Reply:
@lucabergamini

Reply:
Let me try to read it, if that doesn't succeed in an hour I would keep it for another time